+Base/pref_handler.m

```diff
@@ -179,8 +199,41 @@ function set_meta_pref(obj,name,pref)
             val.auto_generated = true;
             val.property_name = name;
         end
-        
+
+    end
+    methods(Access = private)
+        function prop_listener_ctrl(obj,name,enabled)
```

Comment:
Improve speed. Need to remove debug comments from this file. How does one do this on a pull request?

Reply:
you can directly submit more commits to this branch and they will be included

Reply:
+1